docker-compose.yml

```diff
@@ -40,15 +40,15 @@ x-environment: &default-environment
   # Local development route used in Lagoon images and Pygmy to route requests.
   LAGOON_ROUTE: *default-url
   # Local database host (not used in production).
-  MARIADB_HOST: mariadb
+  DATABASE_HOST: database
   # Local database name (not used in production).
-  MARIADB_DATABASE: lagoon
+  DATABASE_NAME: drupal
   # Local database user (not used in production).
-  MARIADB_USERNAME: lagoon
+  DATABASE_USERNAME: drupal
   # Local database password (not used in production).
-  MARIADB_PASSWORD: lagoon
+  DATABASE_PASSWORD: drupal
```

Comment:
These may be non-standard variables in Lagoon, meaning this can break in Lagoon.